firstvoices/backend/management/commands/copy_site.py

```diff
@@ -0,0 +1,581 @@
+    audio_map = {}
+
+    audio_files = list(Audio.objects.filter(site=source_site))
+    for audio_file in audio_files:
+        try:
+            target_file = File(
+                content=UploadedFile(audio_file.original.content.file),
+                site=target_site,
+            )
+            target_file.save()
+            audio_file.original = target_file
+
+            # Speakers
+            current_speakers = list(audio_file.speakers.all())
+            updated_speakers = []
+            for person in current_speakers:
+                person.id = uuid.uuid4()
+                person.site = target_site
+                person.save()
+                updated_speakers.append(person)
+
+            audio_file.site = target_site
+
+            source_audio_id = audio_file.id
+            target_audio_id = uuid.uuid4()
+            audio_map[source_audio_id] = target_audio_id
+
+            audio_file.id = target_audio_id
+            # To circumvent checks added to media models to prevent modification of original file
+            audio_file._state.adding = True
+            audio_file.save()
+
+            audio_file.speakers.set(updated_speakers)
+        except Exception as e:
+            logger.warning(
+                f"Couldn't copy audio file with id: {audio_file.id}", exc_info=e
+            )
+
+    # Copy over rest of the people who are not attached as speakers to any audio
+    person_list = list(Person.objects.filter(site=source_site, audio_set__isnull=True))
+    for person in person_list:
+        person.id = uuid.uuid4()
+        person.site = target_site
+        person.save()
+
+    return audio_map
+
+
+def copy_images_and_return_map(source_site, target_site, logger):
+    image_map = {}
+
+    image_files = list(Image.objects.filter(site=source_site))
+    for image_file in image_files:
+        try:
+            # Content
+            target_file = ImageFile(
+                content=UploadedFile(image_file.original.content.file),
+                site=target_site,
+            )
+            target_file.save()
+            image_file.original = target_file
+
+            image_file.thumbnail = None
+            image_file.small = None
+            image_file.medium = None
+            image_file.site = target_site
+
+            source_img_id = image_file.id
+            target_img_id = uuid.uuid4()
+            image_map[source_img_id] = target_img_id
+
+            image_file.id = target_img_id
+            image_file._state.adding = True
+            image_file.save()
+        except Exception as e:
+            logger.warning(
+                f"Couldn't copy image file with id: {image_file.id}", exc_info=e
+            )
+
+    return image_map
+
+
+def copy_videos_and_return_map(source_site, target_site, logger):
+    video_map = {}
+
+    video_files = list(Video.objects.filter(site=source_site))
+    for video_file in video_files:
```

Comment:
[minor] same as above, for clarity I think "video_file" should be "video_model" so they don't get confused with the content of a video model